server/routerlicious/packages/routerlicious-base/src/riddler/tenantManager.ts

```diff
@@ -455,4 +558,44 @@ export class TenantManager {
         const accessInfo = JSON.parse(this.secretManager.decryptSecret(encryptedAccessInfo));
         return accessInfo;
     }
+
+    private decryptCachedKeys(cachedKey: string) {
+        const keys = JSON.parse(cachedKey);
+        return keys.key2 === "" ? {
+                key1: this.secretManager.decryptSecret(keys.key1),
+                key2: "",
+            } : {
+                key1: this.secretManager.decryptSecret(keys.key1),
+                key2: this.secretManager.decryptSecret(keys.key2),
+            };
+    }
+
+    private async getCacheKey(tenantId: string): Promise<string> {
```

Comment:
nit: would `getKeyFromCache` or even `getCachedKey` be more appropriate? Feel free to dismiss this.

I'm probably getting overly confused with normal language but `getCacheKey` made me think this meant "get the key of this cache", like if the cache had a key property or something like that

Reply:
I understand how this will be confusing. I can rename it to something that is clearer